### ISSUE TITLE
Add jubjub keys

### ISSUE THREAD

**Issue by alv-around** [ZK-Plus/ZnaKes#3]
This ticket is an extension of https://github.com/ZoKratesPlus/ZoKrates/issues/1.
In order to proper test the implementation of the jubjub signatures in ZoKrates, you first to generate valid signatures in the first place. Currently there no maintained libraries that support the key generation of the jubjub curve. Reason for pycrypto to support it.

Pycrypto:

- [x] Generate JubJub keys
- [x] Write tests
- [x] Modify Point and FQ interfaces to use multiple Edwards Curves

**Comment by alv-around** [comment]
@JonathanHeiss I finished this ticket last week, but I am not sure how to proceed with the review. Originally, I though to create a PR to against Zokrate's Pycrypto and have them review it. But last week we discussed, that we would like to use this features as way to position Zokrates+, so for the moment I just create the PR against our `main` and added you as reviewer. Let me know if I should do otherwise.
